packages/fdc3-get-agent/src/strategies/TimeoutLoader.ts

```diff
@@ -26,9 +26,9 @@ export class TimeoutLoader implements Loader {
         this.done = true;
     }
 
-    get(params: GetAgentParams): Promise<DesktopAgent | void> {
+    get({timeoutMs = 0}: GetAgentParams): Promise<DesktopAgent | void> {
```

Comment:
There is a constant for the default timeout elsewhere - but I'm also eliminating the timeoutLoader in another branch (with id validation moving into the loader, loaders can do some ops outside of the timeout so they need to manage the timeouts themselves